sources/rest_api/auth.py

```diff
@@ -33,8 +38,15 @@ class BearerTokenAuth(AuthConfigBase):
     scheme: Literal["bearer"] = "bearer"
     token: TSecretStrValue
 
-    def __init__(self, token: TSecretStrValue) -> None:
```

Comment:
OK I see we have to do that because we have this weird constructor in `CredentialsConfiguration` which takes native value as input. I want to get rid of that... So we'll add those again soon. Maybe you could leave them commented?